mediaflow_proxy/extractors/dlhd.py

```
@@ -1,588 +1,323 @@
+        })
+        
+        from mediaflow_proxy.utils.http_utils import create_httpx_client
+        try:
+            async with create_httpx_client(verify=False) as client:
+                auth_resp = await client.post(auth_url, data=form_data, headers=auth_headers, timeout=12)
+                auth_resp.raise_for_status()
+                auth_data = auth_resp.json()
+                if not auth_data.get("valid"):
+                    raise ExtractorError(f"Initial auth failed with response: {auth_data}")
+            logger.info("New auth flow: Initial auth successful.")
+        except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)